Cargo.toml

```diff
@@ -90,3 +90,6 @@ members = [
 exclude = [
 	"wireguard-go-rust-wrapper"
 ]
+
+[patch.crates-io]
+tokio-rustls = { git = 'https://github.com/rustls/tokio-rustls', branch = 'main' }
```

Comment:
Maybe it would be better to provide a commit hash to avoid fetching a different version in the future?

Reply:
cargo deny will throw an error if that happens though

Reply:
I do not think that this `patch.crates-io` entry is needed :thinking: